ovos_yaml_editor/__init__.py

```
@@ -340,6 +342,11 @@ async def reset_config_post(request: Request, credentials: HTTPBasicCredentials
         return {"success": False, "error": f"Failed to save config: {e}"}
 
 
+@app.post("/status")
+async def status(request: Request):
```

Ruff (0.14.8)

Unused function argument: `request`

(ARG001)